lib/srv/reexec_linux.go

```diff
@@ -30,6 +69,10 @@ func reexecCommandOSTweaks(cmd *exec.Cmd) {
 	// cleaning up child processes, send a signal for graceful shutdown
 	// to children.
 	cmd.SysProcAttr.Pdeathsig = syscall.SIGQUIT
+
+	if reexecPath != "" {
+		cmd.Path = reexecPath
```

Comment:
would be nice to leave a basic comment on why we do this, ie so it still works if the binary gets deleted